### DISCUSSION TITLE
ようこそ HoaryFox のディスカッションへ！

### DISCUSSION THREAD

**Discussion by hrntsm** [hrntsm/HoaryFox#57] [General]
# Welcome!!

私たちは、コミュニティの他のメンバーとつながる場所としてディスカッションを使用しています。ぜひご利用ください。

- 気になることを質問してください。
- アイデアを共有してください。
- 他のコミュニティメンバーとの交流
- 他のメンバーを歓迎し、心を開いてください。ここは私たちのコミュニティであることを忘れないでください。一緒に作りましょう 💪.

**Comment by marusin1235** [comment]
HIRON　様
早速のご返事、ありがとうございます。
「Grasshopper Loading Error」で以下のようなERRORが出ます。
Object: FrameBuilderByAngle (level 1)
{
  呼び出しのターゲットが例外をスローしました。
  TargetInvocationException
}
Object: FrameBuilderByAngle (level 2)
{
  ファイルまたはアセンブリ 'karambaGH, Version=1.15.0.0, Culture=neutral, PublicKeyToken=null'、またはその依存関係の 1 つが読み込めませんでした。指定されたファイルが見つかりません。
  FileNotFoundException
}
Object: NodeBuilder (level 1)
{
  呼び出しのターゲットが例外をスローしました。
  TargetInvocationException
}
Object: NodeBuilder (level 2)
{
  ファイルまたはアセンブリ 'karambaGH, Version=1.15.0.0, Culture=neutral, PublicKeyToken=null'、またはその依存関係の 1 つが読み込めませんでした。指定されたファイルが見つかりません。
  FileNotFoundException
}
対策のご教授、お願いいたします。
